doc/src/fix_ave_moments.rst

```diff
@@ -0,0 +1,286 @@
+can be specified with a wildcard asterisk to effectively specify
+multiple values.
+
+If a value begins with "f\_", a fix ID must follow which has been
+previously defined in the input script.  If no bracketed term is
+appended, the global scalar calculated by the fix is used.  If a
+bracketed term is appended, the Ith element of the global vector
+calculated by the fix is used.  See the discussion above for how I can
+be specified with a wildcard asterisk to effectively specify multiple
+values.
+
+Note that some fixes only produce their values on certain time steps,
+which must be compatible with *Nevery*, else an error will result.
+Users can also write code for their own fix styles and :doc:`add them to
+LAMMPS <Modify>`.
+
+If a value begins with "v\_", a variable name must follow which has been
+previously defined in the input script. Only equal-style or vector-style
+variables can be used, which both produce global values.  Vector-style
+variables require a bracketed term to specify the Ith element of the
+vector calculated by the variable.
+
+Note that variables of style *equal* and *vector* define a formula which
+can reference individual atom properties or thermodynamic keywords, or
+they can invoke other computes, fixes, or variables when they are
+evaluated, so this is a very general means of specifying quantities to
+time average.
+
+----------
+
+The moments are output in the order requested in the arguments following
+the last input.  Any number and order of moments can be specified,
+although it does not make much sense to specify the same moment multiple
+times.  All moments are computed in terms of corrected sample (not
+population) cumulants :math:`k_{1..4}` (see :ref:`(Cramer)<Cramer1>`),
+the standardized moments follow :ref:`(Joanes)<Joanes1>`.
+
+For *mean*, the arithmetic mean :math:`\bar{x} = \frac{1}{n}
+\sum_{i=1}^{n} x_i` is calculated.
+
+For *variance*, the Bessel-corrected sample variance :math:`var = k_2 =
+\frac{1}{n - 1} \sum_{i=1}^{n} (x_i - \bar{x})^2` is calculated.
+
+For *stddev*, the Bessel-corrected sample standard deviation
+:math:`stddev = \sqrt{k_2}` is calculated.
+
+For *skew*, the adjusted Fisher--Pearson standardized moment :math:`G_1
+= \frac{k_3}{k_2^{3/2}} = \frac{k_3}{stddev^3}` is calculated.
+
+For *kurtosis*, the adjusted Fisher--Pearson standardized moment
+:math:`G_2 = \frac{k_4}{k_2^2}` is calculated.
+
+----------
+
+Fix invocation and output can be modified by optional keywords.
+
+The *start* keyword specifies that the first invocation should be no
+earlier than the step number given (but will still occur on a multiple
+of *Nfreq*).  The default is step 0.  Often input values can be 0.0 at
+time 0, so setting *start* to a larger value can avoid including a 0.0
+in a longer series.
+
+The *history* allows keeping a record of previous results.  By default,
+only the most recent invocation is accessible.
+
+For example, this will output values which are delayed by 10
```

Comment:
I think I know what you mean but I don't think "delayed by 10 invocations" is the right terminology.

... this will store output values for the previous 10 Nfreq timesteps, i.e. a window of 10000 steps.